lte/gateway/python/magma/pipelined/app/enforcement_stats.py

```diff
@@ -566,6 +567,20 @@ def _get_rule_id(self, flow):
             self.logger.error('Could not find rule id for num %d: %s',
                               rule_num, e)
             return ""
+    def get_stats(self, cookie: int = 0, cookie_mask: int = 0):
+        #invoke RYU API
+        _, parser = self._datapath.ofproto, self._datapath.ofproto_parser
+        message = parser.OFPFlowStatsRequest(datapath=self._datapath, cookie = cookie, cookie_mask = cookie_mask)
+        response = ofctl_api.send_msg(self, message, reply_cls=parser.OFPFlowStatsReply,
```

Comment:
I think this can throw exceptions (check the api) surround with try/catch and let's handle thosr